migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Can we add some things to this now rather than later? I think that would speed up the process of us all getting this set up on our individual machines.

Reply:
done